### DISCUSSION TITLE
2011/03/30/%E5%96%9C%E6%AC%A2%E8%81%8A%E5%A4%A9%E7%9A%84%E8%B1%86%E8%B1%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#321] [Announcements]
# 2011/03/30/%E5%96%9C%E6%AC%A2%E8%81%8A%E5%A4%A9%E7%9A%84%E8%B1%86%E8%B1%86

豆豆最近不像前两个月那么爱学习了，那时候我对着他说啥，他就学着我说啥。他现在更多的时候不是学我的声音，而更像是在和我对话。他虽然早就已经清楚的发出过“爸”这个音了，但是平时说的很少，更多的时候是我朝他叫“爸”，他就“哎”得答应一声:(

https://blog.qizhen.xyz/2011/03/30/%E5%96%9C%E6%AC%A2%E8%81%8A%E5%A4%A9%E7%9A%84%E8%B1%86%E8%B1%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

lizzie
2011年04月2日 10:30 编辑
宝宝是有很强的自愈能力的哟
我家鹏鹏也是三个月时爱说话，四个月时又不怎么爱说了，现在5个多月又爱说话了哈

回复
